### PR TITLE
improve note

### PR DESCRIPTION
**Pull request by get-itips**
@dariomws thoughts?

closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2949

### CODE REVIEW

**Review by dariomws — APPROVED** [review]

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 1]
```diff
@@ -83,9 +83,10 @@ On the Exchange server, run the following command to synchronize mail-enabled pu
 
 Where `Credential` is your Microsoft 365 or Office 365 admin username and password, and `CsvSummaryFile` is the path to where you would like to log synchronization operations and errors, in .csv format.
 
+> [!IMPORTANT]
+> Before running the script, we recommend that you first simulate the actions that the script would take in your environment by running it as described above with the `-WhatIf` switch. As part of the sync operation, the script, when appropriate, could create, update or delete mail-enabled public folder objects on Exchange Online.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:87]:
```suggestion
> Before running the script, we recommend that you first simulate the actions that the script would take in your environment by running it as described above with the `-WhatIf` switch. As part of the sync operation, the script, when appropriate, could create, update, or delete mail-enabled public folder objects on Exchange Online.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**